src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -71,7 +71,7 @@ private void SetUpTelemetryCallbacks()
                 {
                     requestTraceContext.Stop();
 
-                    var requestTelemetry = requestTraceContext.CreateRequestTelemetry();
+                    var requestTelemetry = requestTraceContext.CreateRequestTelemetry(this.nameResolver);
```

Comment:
I'm not positive, but it looks like this method may be called multiple times. Since the name resolver would have to fetch the name every time this is called, it may be more efficient to fetch the website name once and pass that string into this method instead.